### PR TITLE
[Automanage]

### PR DESCRIPTION
**Pull request by calvinhzy**
---
Automanage support Fix https://github.com/Azure/azure-cli/issues/23552
This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repository and upgrade the version in the pull request but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Automanage

File: src/automanage/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Azure CLI Automanage Extension #
+This is an extension to Azure CLI to manage Automanage resources.
+
+## How to use ##
+Please add commands usage here.
```
Comment (necusjz) [inline src/automanage/README.md:5]: Need to add some usage scenarios.

Comment (calvinhzy) [reply src/automanage/README.md]: Done

**Review by necusjz — APPROVED** [review]

File: src/automanage/azext_automanage/aaz/latest/automanage/configuration_profile/_create.py [hunk 2]
```diff
@@ -0,0 +1,264 @@
...
+
+
+@register_command(
+    "automanage configuration-profile create",
+)
+class Create(AAZCommand):
+    """Create a configuration profile
+
+    :example: create configuration-profile
+        az automanage configuration-profile create -n {profile_name} -g {rg} --configuration {"Antimalware/Enable":false,"Backup/Enable":false,"VMInsights/Enable":true,"AzureSecurityCenter/Enable":true,"UpdateManagement/Enable":true,"ChangeTrackingAndInventory/Enable":true,"GuestConfiguration/Enable":true,"LogAnalytics/Enable":true,"BootDiagnostics/Enable":true}
```
Comment (kairu-ms) [inline src/automanage/azext_automanage/aaz/latest/automanage/configuration_profile/_create.py:21]: The value of `--configuration` is not a valid shorthand syntax expression. We use double quotes to wrap the whole value and use the single quotes to wrap special string in value. I think it should be like this
```
"{'Antimalware/Enable':false,'Backup/Enable':false}"
```

**Review by kairu-ms — APPROVED** [review]

File: src/automanage/azext_automanage/aaz/latest/automanage/configuration_profile/_create.py (hunk 2 above)
Comment (kairu-ms) [reply src/automanage/azext_automanage/aaz/latest/automanage/configuration_profile/_create.py:21]: It's my bad. Freeform dict doesn't support shorthand syntax. Your example is correct.

Comment (calvinhzy) [reply src/automanage/azext_automanage/aaz/latest/automanage/configuration_profile/_create.py:21]: Agreed for shorthand syntax. This `--configuration` param uses freeformdict which is accepting a json so we can keep this example.

**Pull request merged**